keyboards/handwired/yamsek/readme.md

```diff
@@ -0,0 +1,18 @@
+![Yet another Modular split ergonomic Keybard](https://cdn.hackaday.io/images/8060711492102546379.jpg)
```

Comment:
This image should be rehosted on Imgur, and the rest of this file should follow [`data/templates/keyboard/readme.md`](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md).